packages/openapi/api/definitions/devices.yaml

```
@@ -0,0 +1,162 @@
+                  status:
+                    type: number
+                    example: 200
+                  result:
+                    type: array
+                    items:
+                      type: string
+                    example: ['47', '27']
+  device:
+    get:
+      operationId: device
+      tags:
+        - devices
+      summary: Gets a specified peripheral device.
+      parameters:
+        - name: deviceId
+          in: path
+          description: Requested device.
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Command successfully handled - returns the peripheral device
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: number
+                    example: 200
+                  result:
+                    type: object
+                    properties:
+                      id:
+                        type: string
+                        description: Device Id.
+                        example: 47
+                      name:
+                        type: string
+                        description: Device Name.
+                        example: 'PlayoutGateway0'
+                      status:
+                        type: string
+                        enum: [unknown, good, warning_major, warning_minor, bad, fatal]
+                        description: Device status.
+                        example: warning_major
+                      messages:
+                        type: array
+                        items:
+                          type: string
+                          example: Device failed to start
+                        description: Service messages from the device.
+                      deviceType:
+                        type: string
+                        enum:
+                          [
+                            unknown,
+                            mos,
+                            spreadsheet,
+                            inews,
+                            playout,
+                            media_manager,
+                            package_manager,
+                            live_status,
+                            input,
+                          ]
+                        description: Device type.
+                        example: playout
+                      connected:
+                        type: boolean
+                        description: Whether device is currently connected.
+                        example: true
+                    required:
+                      - id
+                      - name
+                      - status
+                      - messages
+                      - deviceType
+                      - connected
+        404:
+          description: The requested device does not exist.
+  deviceAction:
+    put:
```

Should this be a `PUT` or a `POST`? https://restfulapi.net/rest-api-design-tutorial-with-example/ says that `PUT` should be idempotent while `POST` is not. I guess, generally, executing actions on devices is not?
```suggestion
    post:
```